### PR TITLE
feat(Brainly): add presence

### LINKED ISSUE #5439: Brainly | brainly.com
### Discussed in https://github.com/PreMiD/Presences/discussions/4487

<div type='discussions-op-text'>

<sup>Originally posted by **Isa27-0** September  3, 2020</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
   * Contains subscription, but the site is not paid.
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 

**What's the name of the service?** Brainly:  https://brainly.com

**What should the Presence display?** Service name, the question you’re looking at and the length of time you’re on the page

**If possible, please provide a logo for the service (512x512 minimum)** 
_The icon is from the mobile version_
unnamed
</div>

### PR DESCRIPTION
**Pull request by suryomujahid**
**Describe the changes in this pull request:**
Resolves #5439 and includes brainly.co.id, brainly.in, and brainly.ro
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

<br>

</details>

### CODE REVIEW

**Comment by xn-oah** [comment]
https://i.imgur.com/fP9yIAo.png
I think a logo like this might look better tbh, but idk.

Also, shouldn't the color of the presence be #000000 since the majority of the logo uses that color?

**Review by Bas950 — DISMISSED** [review]

File: websites/B/Brainly/presence.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		presenceData.details = "Editing profile";
+	else if (page.includes("/messages"))
+		presenceData.details = "On messages page";
+
+	if (
+		document.querySelector("div.brn-attachment-grabber-container > textarea") &&
+		(
+			document.querySelector(
+				"div.brn-attachment-grabber-container > textarea"
+			) as HTMLInputElement
```
Comment (EncryptedDev) [inline websites/B/Brainly/presence.ts:117]:
```suggestion
			document.querySelector<HTMLInputElement>(
				"div.brn-attachment-grabber-container > textarea"
			) 
```

**Review by EncryptedDev — APPROVED** [review]
Use type generic for all your querySelectors. For example `document.querySelector<HTMLInputElement>("foo bar")`

File: websites/B/Brainly/presence.ts (hunk 1 above)
Comment (EncryptedDev) [reply websites/B/Brainly/presence.ts:117]: This change please

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/B/Brainly/presence.ts (hunk 1 above)
Comment (suryomujahid) [reply websites/B/Brainly/presence.ts]: Done, I use HTMLTextAreaElement instead

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**